flink-catalog-aws/flink-catalog-aws-glue/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,7 @@
+flink-catalog-aws-glue
+Copyright 2014-2023 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
```

Comment:
Is this file needed? Unless we bundle and relocate any dependencies I don't think we need it.